pkg/actions/commands.go

```diff
@@ -95,6 +101,7 @@ func Commands() {
 						cli.StringFlag{Name: "path, p", Usage: "the path to the project", Required: true},
 						cli.StringFlag{Name: "id, i", Usage: "the project id", Required: true},
 						cli.StringFlag{Name: "time, t", Usage: "time of the last sync for the given project", Required: true},
+						cli.StringFlag{Name: "username,u", Usage: "Account Username", Required: false},
```

Comment:
could u be confused with url?

Reply:
Also, `Account Username` not in keeping with format of other Usage statements.

Reply:
* CWCTL already uses username and its alias "u" in other a few other commands
* For the URL flag we seem to have standardised on "url" without an alias apart from 
project create which has both